### PR TITLE
Feat/invitation-preview

### PR DESCRIPTION
**Pull request by Homoakin619**
## Description

- I implemented the design for the email invitation preview
- I also implemented all the modals design for the invitation page
- I implemented the close functionality for all the modals
- Implementations can be found at 
```json 
* BASE_URL/dashboard/invitations/edit
* BASE_URL/dashboard/invitations/preview
```

## Related Issue (Link to linear ticket or Github issue)

Fixes #1234 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Phingaz** [comment]
Bruh I can't find you on slack first lol.
I'll leave a couple of comments for you here, you can hit me up on slack if you have questions

File: app/dashboard/(assets)/applestore.svg
Comment (Phingaz) [inline app/dashboard/(assets)/applestore.svg:1]: Why do you need to upload this

File: app/dashboard/(assets)/dream_logo.svg
Comment (Phingaz) [inline app/dashboard/(assets)/dream_logo.svg:1]: We arent using this logo

File: app/dashboard/(assets)/playstore.svg
Comment (Phingaz) [inline app/dashboard/(assets)/playstore.svg:1]: Why do you need to upload this

File: app/faq/(components)/FaqItems.tsx
Comment (Phingaz) [inline app/faq/(components)/FaqItems.tsx:1]: Why do you need to upload this

File: app/faq/page.tsx
Comment (Phingaz) [inline app/faq/page.tsx:1]: Is the change here necessary?

File: app/globals.css
Comment (Phingaz) [inline app/globals.css:1]: Is the change here necessary?

Comment (Phingaz) [reply app/globals.css:1]: It cant be done on tailwind?

File: public/assets/faq_banner.png
Comment (Phingaz) [inline public/assets/faq_banner.png:1]: Why do you need to upload this

File: public/js/faq.js
Comment (Phingaz) [inline public/js/faq.js:1]: Why do you need to upload this

**Review by Phingaz — CHANGES_REQUESTED** [review]

File: app/dashboard/(assets)/applestore.svg
Comment (Homoakin619) [reply app/dashboard/(assets)/applestore.svg:1]: It was part of the design

File: app/dashboard/(assets)/dream_logo.svg
Comment (Homoakin619) [reply app/dashboard/(assets)/dream_logo.svg:1]: That's the logo used in the design, besides, that's no problem I wil find the current one and change it

File: app/dashboard/(assets)/playstore.svg
Comment (Homoakin619) [reply app/dashboard/(assets)/playstore.svg:1]: Same as above, It was included in the design

File: app/faq/(components)/FaqItems.tsx
Comment (Homoakin619) [reply app/faq/(components)/FaqItems.tsx:1]: Yeah, I forgot to delete this, as mentioned in the group, I had worked on the faq before finding out someone completed it already. Not to worry, I will take it out

File: app/faq/page.tsx
Comment (Homoakin619) [reply app/faq/page.tsx:1]: When I pulled I had some conflicts and had to fix them manually. Forgot to delete the previous faq before pulling

File: app/globals.css
Comment (Homoakin619) [reply app/globals.css:1]: It won't be needed anymore since someone else did the faq

File: public/assets/faq_banner.png
Comment (Homoakin619) [reply public/assets/faq_banner.png:1]: same reason as above (faq)

File: public/js/faq.js
Comment (Homoakin619) [reply public/js/faq.js:1]: same reason as above (faq)

**Review by Phingaz — APPROVED** [review]

**Pull request merged**